.buildkite/hooks/post-command

```diff
@@ -12,7 +12,11 @@
 # upload them as artifacts and delete them afterwards.
 set -e
 if [ -d "/tmp/artifacts/test-summaries" ] && [ "$(ls -A /tmp/artifacts/test-summaries)" ]; then
-    cat /tmp/artifacts/test-summaries/*.txt | buildkite-agent annotate --job "${BUILDKITE_JOB_ID}" --append --style error --context "${BUILDKITE_JOB_ID}"
+    # Only upload annotations if there are at least 2 files in the directory:
+    # 1 header and 1 failed test.
+    if [ "$(find /tmp/artifacts/test-summaries -maxdepth 1 -name '*.txt' | wc -l)" -ge 2 ]; then
```

Comment:
Should it be equals to 3? Bazel run test at most three times. Only mark fail if all three failed.

Reply:
Ah the failing tests actually overwrite the files. The logic is as follows:

1. If a test fails, create directory if it doesn't exist
2. If header doesn't exist, create header file
3. Create summary file for this test, potentially overwriting previous existing file
4. If the test then succeeds in a second or third trial, remove the summary test file

Thus, when a test succeeds after it failed, the summary is always removed. That works and that is also the reason why we currently see these "empty" summaries: The test summary files have all been removed, but the header still exists, so the header is printed.

With this PR, we will only upload annotations if the header and at least one test summary file are present (so 2 files).